x-pack/solutions/observability/packages/kbn-evals-suite-obs-ai-assistant/evals/machine_learning/machine_learning.spec.ts

```diff
@@ -0,0 +1,488 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { MlGetJobsResponse } from '@elastic/elasticsearch/lib/api/types';
+import {
+  cleanupMachineLearningJobs,
+  createApmJobWithNormalData,
+  createAnomalyDetectionJobWithNoData,
+} from '../../src/data_generators/machine_learning_jobs';
+import { createSampleDataMLJobs } from '../../src/data_generators/load_sample_data';
+import { evaluate } from '../../src/evaluate';
+
+/**
+ * NOTE: This scenario has been migrated from the legacy evaluation framework.
+ * - x-pack/solutions/observability/plugins/observability_ai_assistant_app/scripts/evaluation/scenarios/machine_learning/index.spec.ts
+ * Any changes should be made in both places until the legacy evaluation framework is removed.
+ */
+
+evaluate.describe('Machine learning', { tag: '@svlOblt' }, () => {
+  const TEST_INDEX = 'my-index-000001';
+  const TEST_JOB_ID = 'test-job-anomaly-detection';
+  const TEST_DATAFEED_ID = `datafeed-${TEST_JOB_ID}`;
+  const SERVICE_TEST_NAME = 'web-api';
+  const TEST_JOB_ID_2 = 'response_time_anomaly_detection';
+  const TEST_DATAFEED_ID_2 = `datafeed-${TEST_JOB_ID_2}`;
+  let jobs: MlGetJobsResponse = { count: 0, jobs: [] };
+  let jobIds: string[] = [];
+
+  evaluate.beforeAll(async ({ apmSynthtraceEsClient, kbnClient, esClient, log }) => {
+    await cleanupMachineLearningJobs({ esClient, log });
+    await esClient.indices.delete({ index: TEST_INDEX, ignore_unavailable: true });
+    await apmSynthtraceEsClient.clean();
+    await createSampleDataMLJobs({
+      log,
+      kbnClient,
+      sampleDataId: 'logs',
+    });
+
+    await createApmJobWithNormalData(
+      esClient,
+      apmSynthtraceEsClient,
+      SERVICE_TEST_NAME,
+      TEST_JOB_ID,
+      TEST_DATAFEED_ID,
+      log
+    );
+
+    await createAnomalyDetectionJobWithNoData(
+      esClient,
+      TEST_INDEX,
+      TEST_JOB_ID_2,
+      TEST_DATAFEED_ID_2,
+      log
+    );
+
+    await esClient.ml.closeJob({ job_id: TEST_JOB_ID_2, force: true });
+
+    jobs = await esClient.ml.getJobs();
+    jobIds = jobs.jobs.map((job) => job.job_id);
```

Comment:
Should this be called:
```suggestion
    openJobs = await esClient.ml.getJobs();
    openJobIds = jobs.jobs.map((job) => job.job_id);
```

?